### DISCUSSION TITLE
Searchable

### DISCUSSION THREAD

**Discussion by leonliflipp** [thoughtspot/cs_tools#106] [Q&A] [answered]
Hi,

I was trying to run the _searchable gather_ command but I ran into an error. I didn't have this error before whenever I ran it so I was wondering what the reason for this could be. I have posted some screenshots and the log report below. Thanks!

Command: `cs_tools tools searchable gather --syncer csv:///$HOME/Documents/TS_CS_Tools/DEFINITION.toml`

![image](https://github.com/thoughtspot/cs_tools/assets/141184264/e909765d-389b-4539-9d11-89c338a2a9b1)

<img width="507" alt="image" src="https://github.com/thoughtspot/cs_tools/assets/141184264/284fee39-9a70-4218-95e4-34a1d665bd6f">

Log Report: [2024-03-04T14_07_33.log](https://github.com/thoughtspot/cs_tools/files/14486158/2024-03-04T14_07_33.log)

**Comment by boonhapus** [comment]
Hey @leonliflipp ! I think it's related to you having "Disable Pinboard Creation". Can you try removing your user from whatever group has this one?

Or share which cs_tools version you're currently on, I believe the nonstandard privileges should just get warned and flagged in the latest (definitely just warned in the new 1.5.0 which is beta right now).

**Reply by boonhapus** [reply, marked as answer]
If you upgrade to v1.4.16, you should be able to get past this issue. 👍

**Reply by leonliflipp** [reply]
I see, thank you so much!
